coin.lua

```diff
@@ -0,0 +1,24 @@
+Class = require 'lib.class'
+Vector = require 'lib.vector'
+
+Coin = Class {
+	RADIUS = 5
+	objType = 'coin',
+	init = function(self, pos)
+		self.pos = pos
+	end
+}
+
+function Coin:update(dt)
+
+end
+
+function Coin:draw()
+	local r, g, b, a = love.graphics.getColor()
```

Comment:
You can just use `setColor` for `r, g, b`, and not worry about alpha. Also, you probably don't even need to save the color components because no one should ever leave the color to something besides white